test/e2e/tests/multichain/multi-networks-activity.spec.ts

```diff
@@ -0,0 +1,88 @@
+import FixtureBuilder from '../../fixture-builder';
+import {
+  defaultGanacheOptions,
+  openDapp,
+  unlockWallet,
+  WINDOW_TITLES,
+  withFixtures,
+} from '../../helpers';
+import { SMART_CONTRACTS } from '../../seeder/smart-contracts';
+
+const PORT = 8546;
+const CHAIN_ID = 1338;
+const PORT_ONE = 7777;
+const CHAIN_ID_ONE = 1000;
+
+describe('Multichain activity feature', function () {
+  const smartContract = SMART_CONTRACTS.NFTS;
+
+  it('should see activity for current networks and all network', async function () {
```

Comment:
With this E2E; I think we need two transactions, and to check that choosing "All networks" shows 2 items and "Current network" shows 1